.github/workflows/lunaria.yml

```diff
@@ -0,0 +1,44 @@
+name: Lunaria
+
+on:
+  # Trigger the workflow every time a pull request is opened or synchronized at the target `main` branch
+  pull_request_target:
+    types: [opened, synchronize]
+    branches: [main]
+
+# Allow this job to clone the repository and comment on the pull request
+permissions:
+  contents: read
+  pull-requests: write
+
+jobs:
+  lunaria-overview:
+    name: Generate Lunaria Overview
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          # Necessary for Lunaria to work properly
+          # Makes the action clone the entire git history
+          fetch-depth: 0
+
+      - name: Setup PNPM
+        uses: pnpm/action-setup@v3
+
+      - name: Setup Node
+        uses: actions/setup-node@v4
+        with:
+          node-version: 20
+          cache: pnpm
+
+      - name: Install dependencies
+        run: pnpm install
+        shell: bash
+
+      - name: Generate Lunaria Overview
+        uses: yanthomasdev/lunaria-action
```

Comment:
I think we're actually missing the `@main` here to target the branch. Currently, there are no "versions" for the Action since it interacts with the Lunaria version present in the repo itself, it does use some of Lunaria APIs but those are very unlikely to have breaking changes anytime soon.

Reply:
An alternative until the action is versioned could be to use a [specific SHA](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-using-versioned-actions) maybe?

Reply:
We could do this as well if we want to be extra cautious. One thing that makes me opt for a branch is also being able to update with new features without having to update the branch in all the repos

Reply:
Yeah the branch doesn’t buy us much protection from breaking changes or (worst case scenario) dangerous code being pushed to `main` in Lunaria. I think tying to SHA like HiDeoo suggested makes sense here.

It might seem counterintuitive but sometimes some friction can be good here. For example, let’s say you decide Lunaria needs some change that will improve things, but break the current action config — maybe it now needs some new option, or something changed name etc. If everyone is using the unqualified action, you now have a situation where you know pushing that change will break everyone and your only way to avoid that would be to proactively go and 1) lock all users to the current version with PRs, 2) then merge your breaking changes, then finally 3) unlock those users again to upgrade with new PRs.

Following a versioned release flow (or in its absence using a SHA) means users can intentionally upgrade and you can iterate knowing you won’t be causing chaos. (Long term proper semantic versioning is the best of both worlds: breaking change protection PLUS timely updates for safe changes.)

Reply:
Made it a SHA, let me know if we're good

Reply:
Also sharing here if it can helps, as I personally always found the actions [recommended release strategy](https://docs.github.com/en/actions/creating-actions/about-custom-actions#using-release-management-for-actions) using moving git tags to be a bit tedious, this [action](https://github.com/JasonEtco/build-and-tag-action) to help with this by automatically moving the tags.